### PR TITLE
Revamped website.

### PR DESCRIPTION
**Pull request by n1k0**
Call me a nerd but I'm especially glad with this:

r=? @leplatrem @glasserc @almet @Natim @magopian 

### Fixme

- [ ] Find smthg better than *The backend for your next application is already there*
- [x] "Why using kinto" -> "Why use kinto"
- [x] s/Redecentralize/Re-decentralize
- [x] SVG Kinto logo
- [x] move "respect users privacy" underneath "building applications"
- [x]  "across" has one c
- [x] user privacy or users' privacy

### CODE REVIEW

**Comment by n1k0** [comment]
As expected the diff is hard to review, as basically everything has changed. Here's a link to the HTML section, let's focus on texts and wording for now.

File: index.html [hunk 1]
```diff
@@ -1,229 +1,204 @@
...
+        </div>
+
+        <!-- Hero -->
+        <section id="hero" class="container">
+          <header>
+            <h2>The backend for your next application<br>&nbsp;is already there</h2>
+          </header>
+          <p>Kinto is a generic JSON document store with<br>&nbsp;sharing and synchronisation capabilities.</p>
+          <ul class="actions">
+            <li><a href="#why" class="button">Why using Kinto?</a></li>
```
Comment (glasserc) [inline index.html]: Why use Kinto?

File: index.html [hunk 2]
```diff
@@ -1,229 +1,204 @@
...
+      <!-- Header -->
+      <div id="header-wrapper">
+        <div id="header" class="container">
+          <h1 id="logo"><img src="./images/logo.png" alt="Kinto"/></h1>
+        </div>
+
+        <!-- Hero -->
+        <section id="hero" class="container">
+          <header>
+            <h2>The backend for your next application<br>&nbsp;is already there</h2>
```
Comment (glasserc) [inline index.html]: already here. How about "waiting"?

**Comment by n1k0** [comment]
Feedback email from @glasserc:

> This is my review of the design Niko has been working on: http://www.kinto-storage.org/tmp-v2-alt/
>
> "The backend for your next application is already there". This should at least be "already here", but something still strikes me as odd about this and I can't put my finger on what. Maybe it's just me though.
> 
> "Why using kinto" -> "Why use kinto"
> 
> I might change "Redecentralize" into "Re-decentralize", to draw attention to the fact that it's a new word and not "recentralize".
> 
> "It’s hard for frontend developers to respect users privacy when building applications that work offline, store data remotely, and synchronise accross devices." Maybe move "respect users privacy" underneath "building applications" -- "It's hard for frontend developers to build applications that respect user privacy, work offline, store data remotely, and synchronise across devices". (Also, "user privacy" or "users' privacy". Also, "across" has one c.)
> 
> Otherwise everything looks good for me, very polished and professional.

File: index.html [hunk 3]
```diff
@@ -1,229 +1,204 @@
...
+        </section>
+      </div>
+
+      <div class="wrapper" id="why">
+        <div class="container">
+          <article>
+            <header>
+              <h2>Don’t build silos. Own your data. Redecentralise the Web.</h2>
+            </header>
+            <p>It’s hard for frontend developers to respect users privacy when building applications that work offline, store data remotely, and synchronise accross devices. Existing solutions either rely on big corporations that crave user data, or require a non-trivial amount of time and expertise to setup a new server for every new project.</p>
```
Comment (glasserc) [inline index.html]: "user privacy", or "users' privacy". Also, "across".

How about moving "respect users privacy" under "building applications"? "It's hard for frontend developers to build applications that respect user privacy, work offline, store data remotely..." It feels a little strange to have this kind of requirement outside the rest. Are we making a pitch to a philosophy, or to convenience ("easy to set up")? Maybe the current structure is OK if we're trying to argue a philosophical point..

File: index.html [hunk 4]
```diff
@@ -1,229 +1,204 @@
...
+            <li><a href="#why" class="button">Why using Kinto?</a></li>
+          </ul>
+        </section>
+      </div>
+
+      <div class="wrapper" id="why">
+        <div class="container">
+          <article>
+            <header>
+              <h2>Don’t build silos. Own your data. Redecentralise the Web.</h2>
```
Comment (glasserc) [inline index.html]: How about "Re-decentralise"? At first glance, I thought this said "Recentralise".

File: index.html [hunk 5]
```diff
@@ -1,229 +1,204 @@
...
+            </section>
+          </div>
+        </section>
+      </div>
+
+      <!-- Promo -->
+      <div id="promo-wrapper">
+        <section id="promo">
+          <h2>If you know HTTP, you know Kinto</h2>
+          <a href="http://kinto.readthedocs.io/en/latest/tutorials/first-steps.html" class="button">Take the tutorial</a>
```
Comment (glasserc) [inline index.html]: "Take" feels like the wrong verb for a tutorial. "Follow"? "Read"?

File: index.html [hunk 6]
```diff
@@ -1,229 +1,204 @@
...
+                <li><a href="http://kinto.readthedocs.io/en/stable/api/1.x/permissions.html">Permissions</a></li>
+                <li><a href="http://kinto.readthedocs.io/en/stable/tutorials/first-steps.html">Synchronization</a></li>
+                <li><a href="https://kinto.readthedocs.io/en/latest/api/1.x/quotas.html#api-quotas">Quotas</a></li>
+              </ul>
+            </section>
+            <section class="6u 12u(narrower) feature">
+              <header>
+                <h2><i class="fa fa-plug"></i> Pluggable by design</h2>
+              </header>
+              <p>If you miss something, an extensible architecture and many existing plugins probably provide what you need.</p>
```
Comment (glasserc) [inline index.html]: "If something is missing"

File: index.html [hunk 7]
```diff
@@ -1,229 +1,204 @@
...
+          </div>
+        </div>
+      </div>
+
+      <!-- Features 2 -->
+      <div class="wrapper">
+        <section class="container">
+          <header class="major">
+            <h2>Kinto provides all the tools you need to get productive</h2>
+            <p>Web, Native, Server, Kinto is target platform agnostic</p>
```
Comment (glasserc) [inline index.html]: Maybe replace the last comma with an em-dash or a colon?

File: index.html (hunk 2 above)
Comment (n1k0) [inline index.html]: not too fan of it either... we need to find something better :)

**Comment by n1k0** [comment]
Okay i'm gonna land this as is for now, we can keep iterating next.

**Pull request merged**